packages/components/src/Switch.ts

```diff
@@ -0,0 +1,63 @@
+import { ForwardedRef, forwardRef, ReactNode } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import RSwitch, { ReactSwitchProps } from 'react-switch';
+
+import { useThemeFromContext } from './theme';
+
+interface SwitchLabelProps {
+  children?: ReactNode;
```

Comment:
Why are children optional?

Reply:
Force of habit / copying from React's `PropsWithChildren` type. But yeah, in this case, children would always be expected. This could be typed as a required ReactNode or even a required string.

Reply:
Updated in https://github.com/DataBiosphere/terra-ui/pull/4364/commits/542cb95fadfe496afc1f3738639a9bcb856c5897.